manifests/config.pp

```diff
@@ -20,6 +20,16 @@
     group => 'tomcat',
   }
 
+  file { '/etc/candlepin/certs/candlepin-ca.crt':
+    ensure => file,
+    source => $candlepin::ca_cert,
+  }
+
+  file { '/etc/candlepin/certs/candlepin-ca.key':
+    ensure => file,
+    source => $candlepin::ca_key,
```

Comment:
@ekohl I wasn't sure if this was the best way to do this especially thinking ahead to splitting services. The other idea I had was to pass in the contents of the certificate rather than a location. However, I wasn't entirely clear how to read in file contents in one module and then pass it in if the file has not been created yet.

Reply:
I don't think this is a good approach. Now you have a copy of the CA key elsewhere. IMHO it would be better to change puppet-certs to deploy to this location.